src/opnsense/mvc/app/models/OPNsense/BootEnvironments/General.xml

```diff
@@ -0,0 +1,16 @@
+<model>
+    <mount>//OPNsense/BootEnvironments/general</mount>
+    <description>
+        A mesh VPN that makes it easy to connect your devices, wherever they are.
```

Comment:
Probably a typo here?

Reply:
Yes, copied that file from the plugin I'm working on for Tailscale and failed to change the description. Will fix.